app/utils/ParserUtil.scala

```diff
@@ -84,9 +84,8 @@ class ParserUtil @Inject()(val ersUtil: ERSUtil,
   }
 
   def getSheetErrors(schemeErrors: ListBuffer[SheetErrors]): ListBuffer[SheetErrors] = {
-    val errorCount: Int = appConfig.errorCount.getOrElse(HUNDRED)
```

Comment:
I'm fine with removing the default value here so long it was confirmed, this error count value exists in all necessary environments/app-config-base

Reply:
yep - it's not overridden anywhere, in fact